src/components/Layout/Page.tsx

```diff
@@ -15,7 +15,7 @@ import {useRouter} from 'next/router';
 import {SidebarNav} from './SidebarNav';
 import {Footer} from './Footer';
 import {Toc} from './Toc';
-import SocialBanner from '../SocialBanner';
+// import SocialBanner from '../SocialBanner';
```

Comment:
We could remove this instead of commenting out, could always look at history if we need it back